guide/principles/principles.md

```diff
@@ -0,0 +1,182 @@
+**Don't**
+- Exclude people by building features that only work in certain countries
+- Add technical detail that is not required knowledge, or technical terms like seed phrase XPUBs, mnemonics etc.
+- Put all education up front and expect people to read and remember it
+
+---
+
+## Interoperability
+
+> Enable import and export of wallets, maximise backwards compatibility and use of open standards
+
+Bitcoin is an open-source protocol, operated in a decentralized manner. This has led to a number of standards being developed for products building on the protocol to be compatible. It should be easy to switch and move your Bitcoin wallet to a different application, should you wish. Making sure that your product supports as many of these standards as possible is best practice and builds trust. More on [wallet interoperability]({{ '/guide/foundations/wallet-interoperability/' | relative_url }}). 
+
+**Do**
+- Support import and export of wallets
+- Support as many relevant BIPs as possible
+- Be transparent with which ones you do and don’t support
+- Maximize backwards compatibility
+
+**Don't**
+- Lock your users in
+- Implement proprietary solutions when open standards exist
+
+---
+
+## Transparency
+
+> Be open and transparent with how your product works, open-source your code when possible
+
+While an open and decentralized financial system that lets people connect directly with it is great, it does put a lot of burden on the users to pick a product they trust, and that works for them. We can make this easier by freely sharing information about how our products work, what technologies they use and rely on. Let people verify that your claims are true, ideally by open-sourcing your own code. It is also very important to be open about what risks the user might be taking on with self-custody, under which scenarios they might lose access to their funds and best practices for avoiding this. 
+
+**Do**
+- Be open and transparent with how your product works
+- Let people verify your claims by open-sourcing your code when possible
+- Explain what risks the user is taking on, and how best to mitigate them
+
+**Don't**
+- Make claims that are not explained or verifiable 
+
+---
+
+## Privacy
+
+> Minimize collection of personal information, and maximize financial transaction privacy
+
+A common misconception of Bitcoin is that it provides complete anonymity and privacy of transactions. Since the blockchain is an unchangeable ledger of all transactions ever made, it is in fact very hard to not have your complete transaction history visible once even a single one of your addresses is connected to you. If Bitcoin is to become viable for a wider audience and daily use we ought to take privacy seriously. This is certainly not to enable or encourage illicit activity, but to protect individual financial privacy. We would not accept our bank to publish our financial transactions to our Twitter or Facebook feeds, so should avoid that scenario with Bitcoin as well.
```

Comment:
```suggestion
A common misconception of Bitcoin is that it provides complete anonymity and privacy of transactions. Since the blockchain is an unchangeable ledger of all transactions ever made, it is in fact very hard to not have your complete transaction history visible once even a single one of your addresses is connected to you. If Bitcoin is to become viable for a wider audience and daily use we ought to take privacy seriously. This is certainly not to enable or encourage illicit activity, but to protect individual financial privacy. We would not accept our bank to publish our financial transactions to our Twitter or Facebook feeds, so we should avoid that scenario with Bitcoin as well.
```